web-interface/vm-consoles-modules/vm-console-web.adoc

```diff
@@ -1,9 +1,9 @@
 [[vm-console-web]]
 == Virtual Machine Console Sessions
 
-You can connect to the VNC and serial consoles of a running virtual machine from the *Consoles* tab in the *Virtual Machine Details* screen of the web console.
+You can connect to the VNC, Serial, RDP consoles of a running virtual machine from the *Consoles* tab in the *Virtual Machine Details* screen of the web console.
```

Comment:
I'm sorry... I wasn't clear before. You still keep the "and" when you use the Oxford comma.

Reply:
ooh, interesting. Should be fixed now!